src/components/Tooltip/Tooltip.md

```diff
@@ -5,15 +5,17 @@ You should always wrap Tooltip around a tabbable element, ensuring keyboard navi
 
 Basic usage:
 
-```js { "props": { "data-description": "basic" } }
-<Tooltip text="This is the tooltip content.">
-  <Clickable>Hover over me</Clickable>
-</Tooltip>
+```js { "props": { "data-description": "basic", "data-action-states": "[{\"action\":\"hover\",\"selector\":\".y-clickable\"}]"  } }
```

Comment:
can we just use single quotes instead of escaping these?

Reply:
We could have, but then we would have had to replace them with double quotes in `react-styleguidist-visual` since single quote are invalid json.